src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -1782,11 +1782,6 @@
 					marqueeWidget = null;
 
 				self._unbindEvents();
-
-				self._items.forEach(function (li) {
-					self.element.appendChild(li);
-					li.setAttribute("style", "");
-				});
```

Comment:
@lslachciak, @TomaszLukawskiSam,
Do you know why it append items to the element here?